src/Makefile.manual

```diff
@@ -81,7 +81,8 @@ stdlib_io.o: \
 	stdlib_optval.o \
 	stdlib_kinds.o
 stdlib_linalg.o: \
-	stdlib_kinds.o
+	stdlib_kinds.o \
+    stdlib_string_type.o
```

Comment:
Later I will remove this dependency `stdlib_string_type.o` (There is no need here, it is a wrong).